shell/platform/darwin/ios/framework/Source/FlutterPlatformViews_Internal.mm

```diff
@@ -28,7 +28,9 @@
     : layer_pool_(std::make_unique<FlutterPlatformViewLayerPool>(surface_factory)),
       weak_factory_(std::make_unique<fml::WeakPtrFactory<FlutterPlatformViewsController>>(this)){};
 
-FlutterPlatformViewsController::~FlutterPlatformViewsController() = default;
+FlutterPlatformViewsController::~FlutterPlatformViewsController() {
+  [flutter_view_.load() release];
```

Comment:
ah. good catch.

Reply:
Previously the object was contained by a smart pointer, this wasn't needed.  Now that we need to make it atomic I had to add the explicit release.